openwisp_controller/config/views.py

```diff
@@ -73,3 +74,26 @@ def schema(request):
         c = login_required_error
         status = 403
     return HttpResponse(c, status=status, content_type='application/json')
+
+
+def get_template_default_values(request):
+    """
+    returns default_values for one or more templates
+    """
+    pks = []
+    for p in request.GET.get('pks', '').split(','):
+        try:
+            UUID(p, version=4)
+        except ValueError:
+            return JsonResponse(
+                {'error': 'invalid template pks were received'}, status=400
+            )
+        else:
+            pks.append(p)
+    values = Template.objects.filter(pk__in=pks).values_list(
+        'default_values', flat=True
+    )
```

Comment:
does it work if you add `.only('default_values')`? The query should be more efficient and hence faster.

Reply:
from my [brief research](https://stackoverflow.com/questions/11974691/difference-between-values-and-only), values_list is more performant than `only` because it gets just the values of the field while `only` returns the object.